### PR TITLE
CreateAlert support

### PR DESCRIPTION
**Pull request by maxrichie5**
### Background

This contains the changes from https://github.com/panther-labs/panther_analysis_tool/pull/431 and reverts https://github.com/panther-labs/panther_analysis_tool/pull/434

The issue was not with the code but with people's env configuration. They weren't using pipenv which was leading to depedency weirdness. Pip itself wasn't upgrading panther-core as a part of the PAT upgrade because nothing was forcing it to. 

### Changes

* Adds back in the CreateAlert support
* Updates setup.py to have a min version of panther-core

### Testing

I checked that the requires.txt that is generated from setup has the correct version
I installed the built tar ball and ran pat test and it worked

### CODE REVIEW

File: requirements.txt
Comment (maxrichie5) [inline requirements.txt:1]: This was generated from `make reqs`

File: setup.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "schema",
     "semver",
     "sqlfluff",
-    "panther_core",
+    "panther-core>=0.6.7",
```
[the remark this answers is not in the document]
Comment (dekatzenel) [reply setup.py:24]: Do we need to do this with other dependencies, maybe in a separate PR?

**Review by darwayne — COMMENTED** [review]
Nice stuff. Do we need to update anything [here](https://github.com/panther-labs/panther_analysis_tool/blob/main/.github/workflows/test_release_publish.yml) as well?

**Review by nskobov — APPROVED** [review]

**Comment by maxrichie5** [comment]
> Nice stuff. Do we need to update anything [here](https://github.com/panther-labs/panther_analysis_tool/blob/main/.github/workflows/test_release_publish.yml) as well?

I believe [this](https://github.com/panther-labs/panther_analysis_tool/blob/main/.github/workflows/test_release_publish.yml#L29) step covers my change

**Review by darwayne — APPROVED** [review]

**Pull request merged**